TM1py/Services/ElementService.py

```diff
@@ -168,11 +168,65 @@ def get_element_identifiers(self, dimension_name: str, hierarchy_name: str,
              """.format(
             elem_mdx=mdx_element_selection,
             attr_mdx=",".join(build_element_unique_names(
-                ["}ElementAttributes_" + dimension_name] * len(alias_attributes),
-                alias_attributes)),
+                ["}ElementAttributes_" + dimension_name] * len(alias_attributes), alias_attributes)),
             dim=dimension_name)
         return self._retrieve_mdx_rows_and_cell_values_as_string_set(mdx, **kwargs)
 
+    def get_element_by_attribute(self, dimension_name: str, hierarchy_name: str, attribute: str,
```

Comment:
I think the name sounds too much like filtering.

I think a name like `get_attribute_of_elements` would be more indicative.

Reply:
I like this name.